src/index.js

```diff
@@ -94,6 +94,15 @@ window.App = {
     const myNametag = document.querySelector("#player-rig .nametag");
     myNametag.setAttribute("text", "value", username);
 
+    //change spawn position & rotation
+    const myHead = document.querySelector("#player-rig #head");
+    let rotation = Math.random() * 2 * Math.PI;
+    let position = myHead.getAttribute("position");
+    let newPosition = getSpawnPositionInCircle(0, 0, 4, rotation);
+    newPosition.y = position.y;
+    myHead.setAttribute("position", newPosition);
+    myHead.setAttribute("rotation", getRotationToTarget(newPosition, {x:0, y:0, z:0}));
```

Comment:
This logic should really be in a component on the `head` element.

Reply:
it should probably be on the player rig not the head, but we probably want it to end up placing the head above the spawn point we pick (like we do for teleporting)